applications/crossbar/src/modules/cb_system_configs.erl

```diff
@@ -280,6 +284,17 @@ validate_with_parent(Context, ConfigId, Node, Parent) ->
                                      end
                                     ).
 
+-spec validate_request(cb_context:context(), ne_binary(), ne_binary(), kz_json:object()) -> cb_context:context().
+validate_request(Context, Id, Schema, Parent) ->
+    RequestData = strip_id(cb_context:req_data(Context)),
+    FullConfig = kz_json:merge_recursive(strip_id(kz_json:public_fields(Parent)), RequestData),
+    cb_context:validate_request_data(Schema, cb_context:set_req_data(Context, FullConfig),
+                                     fun(Ctx) ->
+                                             Doc = maybe_set_id(Id, kz_json:diff(RequestData, Parent)),
```

Comment:
you should use `cb_context:doc(Ctx)` instead of `RequestData` because it will have the `defaults` applied